pipeline/ingest/behavior.py

```diff
@@ -713,11 +611,6 @@ def make(self, key):
             rows['trial_event'], ignore_extra_fields=True,
             allow_direct_insert=True, skip_duplicates=True)
 
-        log.info('BehaviorIngest.make(): ... CorrectedTrialEvents')
-        BehaviorIngest().CorrectedTrialEvents().insert(
```

Comment:
this + related values were intended to log any adjustments made to the raw input data due to bad gui duration items - we are not using the gui durations now but calculating, but are the 'end' cases true?

Reply:
I am unsure what you mean, which 'end' cases?

Reply:
there are a few blocks removed which added things to 'CorrectedTrialEvents' which are now removed (so nothing is logged to corrected trial events) - I don't recall the specific source cases but it was something about trials without a duration at the 'ends' of a session taking a duration value from previous trials or something like that (in retrospect, should have noted better in the code - that said, git history should have details).

Basically the question is: do things still need to be noted in CorrectedTrialEvents

Reply:
ah, yeah the `CorrectedTrialEvents` I believe was there to notate which event didn't have the right "duration" read from the GUI. Now that we're not using GUI for duration anymore, this becomes obsolete thus removed